src/app/code/community/Alma/Installments/Helper/FeePlansHelper.php

```diff
@@ -322,4 +328,32 @@ public function validateFeePlanMinAndMaxCustomAmount($feePlans)
         return $feePlans;
     }
 
+
+    /**
+     * @return string
+     */
+    public function getEnableFeePlansForBadge()
+    {
+        $plansForBadge = $this->getEnabledFeePlansConfigFromBackOffice();
+        foreach ($plansForBadge as $panKey => $plan) {
+            $plansForBadge[$panKey] = $this->formatPlanForBadge($plan);
```

Comment:
I see that later you use `array_values` to remove the keys.
Why don't you use another array ?
```
$formattedBadgePlans = [];
foreach ($plansForBadge as $plan) {
    $formattedBadgePlans[] = $this->formatPlanForBadge($plan);
}
return json_encode($formattedBadgePlans);
```

Reply:
Done